pkg/http/api_actors.go

```diff
@@ -0,0 +1,174 @@
+				Name: "InvokeActor",
+			},
+		},
+		{
+			Methods:         []string{nethttp.MethodGet},
+			Route:           "actors/{actorType}/{actorId}/state/{key}",
+			Version:         apiVersionV1,
+			Group:           endpointGroupActorV1State,
+			FastHTTPHandler: a.onGetActorState,
+			Settings: endpoints.EndpointSettings{
+				Name: "GetActorState",
+			},
+		},
+		{
+			Methods: []string{nethttp.MethodDelete},
+			Route:   "actors/{actorType}/{actorId}/state/{key}",
+			Version: apiVersionV1,
+			Group:   endpointGroupActorV1State,
+			Handler: a.onDeleteActorStateHandler(),
+			Settings: endpoints.EndpointSettings{
+				Name: "DeleteActorState",
+			},
+		},
+		{
+			Methods:         []string{nethttp.MethodPost, nethttp.MethodPut},
+			Route:           "actors/{actorType}/{actorId}/reminders/{name}",
+			Version:         apiVersionV1,
+			Group:           endpointGroupActorV1Misc,
+			FastHTTPHandler: a.onCreateActorReminder,
+			Settings: endpoints.EndpointSettings{
+				Name: "RegisterActorReminder",
+			},
+		},
+		{
+			Methods:         []string{nethttp.MethodPost, nethttp.MethodPut},
+			Route:           "actors/{actorType}/{actorId}/timers/{name}",
+			Version:         apiVersionV1,
+			Group:           endpointGroupActorV1Misc,
+			FastHTTPHandler: a.onCreateActorTimer,
+			Settings: endpoints.EndpointSettings{
+				Name: "RegisterActorTimer",
+			},
+		},
+		{
+			Methods:         []string{nethttp.MethodDelete},
+			Route:           "actors/{actorType}/{actorId}/reminders/{name}",
+			Version:         apiVersionV1,
+			Group:           endpointGroupActorV1Misc,
+			FastHTTPHandler: a.onDeleteActorReminder,
+			Settings: endpoints.EndpointSettings{
+				Name: "UnregisterActorReminder",
+			},
+		},
+		{
+			Methods:         []string{nethttp.MethodDelete},
+			Route:           "actors/{actorType}/{actorId}/timers/{name}",
+			Version:         apiVersionV1,
+			Group:           endpointGroupActorV1Misc,
+			FastHTTPHandler: a.onDeleteActorTimer,
+			Settings: endpoints.EndpointSettings{
+				Name: "UnregisterActorTimer",
+			},
+		},
+		{
+			Methods:         []string{nethttp.MethodGet},
+			Route:           "actors/{actorType}/{actorId}/reminders/{name}",
+			Version:         apiVersionV1,
+			Group:           endpointGroupActorV1Misc,
+			FastHTTPHandler: a.onGetActorReminder,
+			Settings: endpoints.EndpointSettings{
+				Name: "GetActorReminder",
+			},
+		},
+	}
+}
+
+// Route: DELETE "actors/{actorType}/{actorId}/state/{key}"
+func (a *api) onDeleteActorStateHandler() http.HandlerFunc {
+	return UniversalHTTPHandler(
+		a.universal.DeleteActorState,
+		UniversalHTTPHandlerOpts[*runtimev1pb.DeleteActorStateRequest, *emptypb.Empty]{
+			InModifier: func(r *http.Request, in *runtimev1pb.DeleteActorStateRequest) (*runtimev1pb.DeleteActorStateRequest, error) {
+				in.ActorId = chi.URLParam(r, actorIDParam)
+				in.ActorType = chi.URLParam(r, actorTypeParam)
+				return in, nil
+			},
+			SuccessStatusCode: http.StatusAccepted,
```

Comment:
The "accepted" status code is normally used to indicate that an operation has been accepted and will be executed in background. Here, the operation is already executed synchronously.

Suggest using either 204 (No Content) or 200 (OK)
```suggestion
			SuccessStatusCode: http.StatusNoContent,
```